cmd/incusd/instance_console.go

```diff
@@ -500,6 +521,40 @@ func instanceConsolePost(d *Daemon, r *http.Request) response.Response {
 		return response.BadRequest(fmt.Errorf("Instance is frozen"))
 	}
 
+	// Find any running 'ConsoleShow' operation for the instance.
+	// If the '--force' flag was used, cancel the running operation. Otherwise, notify the user about the operation.
+	for _, op := range operations.Clone() {
+		// Consider only console show operations with Running status.
+		if op.Type() != operationtype.ConsoleShow || op.Project() != projectName || op.Status() != api.Running {
+			continue
+		}
+
+		// Fetch instance name from operation.
+		r := op.Resources()
+		apiUrls := r["instances"]
+		if len(apiUrls) < 1 {
+			return response.SmartError(fmt.Errorf("Operation does not have an instance URL defined"))
+		}
+
+		urlPrefix, instanceName := path.Split(apiUrls[0].URL.Path)
+		if urlPrefix == "" || instanceName == "" {
+			return response.SmartError(fmt.Errorf("Instance URL has incorrect format"))
+		}
+
+		if instanceName != inst.Name() {
+			continue
+		}
+
+		if !post.Force {
+			return response.SmartError(fmt.Errorf("Active session exists. Use --force to force the connection"))
```

Comment:
We shouldn't mention command line flags in server-side errors.

So maybe go with `This console is already connected. Force is required to take it over.`

Reply:
Fixed.